service/controller/appcatalog/v1/resource/index/resource.go

```diff
@@ -19,20 +21,24 @@ type Config struct {
 	// Dependencies.
 	K8sClient kubernetes.Interface
 	Logger    micrologger.Logger
+
+	// Settings.
+	IndexNamespace string
+	ProjectName    string
 }
 
 // Resource implements the index resource.
 type Resource struct {
 	// Dependencies.
 	k8sClient kubernetes.Interface
 	logger    micrologger.Logger
-}
 
-func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
-	return nil, nil
+	// Settings.
+	projectName    string
```

Comment:
@tomahawk28 The ordering still isn't right in multiple places like here. Please can you do a follow up PR so this gets sorted.